crates/context/README.md

```diff
@@ -0,0 +1,65 @@
+# Calimero Contexts
+
+- [Context Lifecycle](#context-lifecycle)
+- [Administrative Operations](#administrative-operations)
+- [Context State](#context-state)
+- [State Mutations](#state-mutations)
+- [Context Membership](#context-membership)
+  - [Invitations](#invitations)
+  - [Leaving](#leaving)
+- [Encryption](#encryption)
+- [Application Upgrade](#application-upgrade)
+- [Context Deletion](#context-deletion)
+
+A context in Calimero is an instance of a deployed application members of which share a synchronized state.
```

Comment:
Can you simplify this intro? 

In terminology we have this `Context is the core of the Calimero ecosystem. It is an application specific network designed to enable direct communication between users, eliminating the need for intermediaries`

More questions than answers arise from this which we should prevent:
- where are members deployed
- are they application members or context members
- is there then a global state that they all share

Reply:
Later it is explained well and pieces come together. But as this is the first sentence all those info is not known. That is only my concern

Reply:
simplified